project_proposals/osrd.md

```diff
@@ -66,4 +66,55 @@ Actually we have an internal process which is the same for internal and external
 ## How can someone become s committer to the project?
 For now, only the maintainers can make the final approval and merge the PR.
 ## How can someone become a maintainer of the project?
-It is not decided yet. I assume that once we have a contributor that provide good work, he or she can be granted maintainer. But we prefer not to have too much maintainers. 
+It is not decided yet. I assume that once we have a contributor that provide good work, he or she can be granted maintainer. But we prefer not to have too much maintainers.
+
+# Stage 2: Incubation
+
+## Healthy metrics
+
+Criteria:
+
+* Have a healthy number of committers. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project. Committers can come from the same organization.
+* Demonstrate a substantial ongoing flow of commits and merged contributions.
+* Since these metrics can vary significantly depending on the type, scope and size of a project, the TC has final judgement over the level of activity that is adequate to meet these criteria.
+
+Proof of meeting criteria:
+
+* Some stats for the last month (only for the main repository).
+  * 36 authors (pushed 239 commits)
+  * 246 active PR (190 merged)
+  * 246 active issues (99 closed)
+  * 887 files touched (+17,018 / -24,855)
+  * 3 releases
+
+> [!NOTE]
+> All this information is taken from [here](https://github.com/OpenRailAssociation/osrd/pulse)
+
+## Open to external contributors
+
+* Have a clear release process which communicates to users what to expect from releases. This includes a clear versioning scheme and cryptographically signing release artifacts.
+  * Describe on our website here: https://osrd.fr/en/docs/guides/release/

```

Comment:
In addition to how the releases work from a technical point of view, it would be helpful to have information how users are affected by releases and how they can plan with releases. This would include things like the release cycle, stability of releases, if and how beta and other preview releases are available, and how this is communicated (e.g. is there a versioning scheme indicating stability of releases).

Related to this is information how users can provide feedback for releases, e.g. by opening issues, or how they can follow or even influence the roadmap, so that they can plan regarding future releases.